spec/models/device_problem_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe DeviceProblem, type: :model do
+
+  it { should belong_to(:device) }
+
+  let!(:device) do
+    Device.create(name: 'Fake Device Name', image_url: 'http://www.fakeimageurl.com/')
+  end
+
+  subject do
+    DeviceProblem.create(
+      problem: 'Fake Problem',
+      content: JSON.parse("{ \"fake_content\": \"content\" }"),
```

Comment:
You can pass a Ruby has into a JSON-type column and it'll handle it correctly.  You don't have to do the extra work of building up a valid JSON string and then parsing it.